test/onnx/model_defs/op_test.py

```diff
@@ -46,3 +46,13 @@ def __init__(self):
     def forward(self, x):
         output = self.features(x)
         return output
+
+class FakeQuantNet(nn.Module):
```

Comment:
Would it be possible to add a model level test in _[test_models_onnxruntime.py](https://github.com/pytorch/pytorch/blob/master/test/onnx/test_models_onnxruntime.py)? It would also serve as documentation of the user scenario for this functionality.

Reply:
Seems test_models_onnxruntime.py imports from test_pytorch_onnx_onnxruntime , https://github.com/pytorch/pytorch/blob/master/test/onnx/test_models_onnxruntime.py#L10

There is a test in test_pytorch_onnx_onnxruntime

Reply:
That is just the `run_model_test` helper method. Also, the test point in  test_pytorch_onnx_onnxruntime is unit test. I was suggesting adding a larger model test in this file. Do you have any model that required this functionality? That may be a good candidate for using for the model-level test.

Reply:
There is a test added to test_models.py and run by test_models_onnxruntime.py. 

Are you suggesting a larger one? Resnet50 feels to big of a test, maybe a LeNet?

Reply:
Yes, we need a larger model test. We actually do have resnet50 as part of test_models.

Reply:
Added a full qat resnet50.